### PR TITLE
Updated images for CVE-2015-4000 (MGASA-2015-0246)

### PR DESCRIPTION
**Pull request by juanluisbaptiste**
Updated for CVE-2015-4000

### CODE REVIEW

**Comment by tianon** [comment]
I see that the commits referenced here did exist at one point (https://github.com/juanluisbaptiste/docker-brew-mageia/commit/9dde5fca0ba75eddab7831b67869f886c4c16e70 and https://github.com/juanluisbaptiste/docker-brew-mageia/commit/5adb71a8270dca00f7e4f071b3c7318dda2a28de), but no branches reference them now, so that's why Travis is failing and why this will fail to build.  I think getting both of those tarballs into that top commit of the "dist" branch there on your repo should be correct, right?

**Comment by juanluisbaptiste** [comment]
Dammit... I don't know what I did while also preparing the mga 5 image I think I deleted these.

**Comment by juanluisbaptiste** [comment]
I don't know what I'm doing wrong, this is what I did, according to the instructions you once gave me when initially importing these images:

git branch -D dist
git checkout -b dist
create images
git add .
git commit
git push
Edit my stackbrew fork with the new commit id
git add
git commit
git push -f
Then I went to my stackbrew fork and clicked on pull request.

**Comment by juanluisbaptiste** [comment]
@tianon can you give me some help ? I really don't know what I'm doing wrong here.

File: library/mageia [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # maintainer: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 
-latest: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 4
-cauldron: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 cauldron
-4: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 4
+latest: git://github.com/juanluisbaptiste/docker-brew-mageia@a814b1bd92e4bd4967ca17c56cd873df67ac4d91 4
```
Comment (tianon) [inline library/mageia]: Your repo looks correct now, but I think Travis is failing on the extra blank space at the end of this line.

Also, shouldn't you be adding `5` in here somewhere? (and should it be the new "`latest`"?)

**Comment by juanluisbaptiste** [comment]
@tianon thanks, about mageia 5, I was going to do it in a separate pull request, to not mix topics, or should I add that commit to this PR ?

**Comment by tianon** [comment]
I certainly don't have a problem with adding it here! :+1:

**Comment by juanluisbaptiste** [comment]
Ok, added !

**Comment by tianon** [comment]
This is failing to build, but I think it's because all the commit references should be `a814b1bd92e4bd4967ca17c56cd873df67ac4d91` (since the one you put in the file here is the one where you added the `Dockerfile`, but the rootfs tarball doesn't exist there).

**Comment by juanluisbaptiste** [comment]
@tianon Fixed.

**Comment by tianon** [comment]
:+1: LGTM

Build test of #840; 4cd89cffc8ce3deb0e6c68902b75d204fb5be153 (`mageia`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/4cd89cffc8ce3deb0e6c68902b75d204fb5be153/library/mageia"
$ bashbrew build "$url"
Processing mageia:latest ...
Processing mageia:cauldron ...
Processing mageia:4 ...
Processing mageia:5 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing mageia:latest
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing mageia:cauldron
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
testing mageia:4
	'utc' [1/3]...passed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...passed
```

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**